lms/templates/api_admin/catalog-results.underscore

```diff
@@ -0,0 +1,6 @@
+<h2 class="api-subheading"><%- gettext("This catalog's courses:") %></h2>
+<ul class="preview-results-list">
```

Comment:
Will the response we get back be paginated?  Do we need to take that into account here?